gravitee-apim-console-webui/src/management/api/documentation-v4/components/api-documentation-v4-content-editor/api-documentation-v4-content-editor.component.scss

```diff
@@ -15,5 +15,16 @@
   &__editor {
     display: grid;
     grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
+    gap: 10px;
   }
 }
+
+:host ::ng-deep table td {
+  word-break: break-all;
+  overflow-wrap: break-word;
+  white-space: normal;
+}
+
+:host ::ng-deep p {
+  word-wrap: break-word;
+}
```

Comment:
Do we do that only for `<markdown` component ?
Have you check into the library documentation if we can pass custom css to the component ? It would be nice to avoid ng-deep as much as possible and it's even more true that here you have pointing directly the table and not a specific class.